src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using static AspNet.Security.OAuth.Alipay.AlipayAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Alipay
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="AlipayAuthenticationHandler"/>.
+    /// </summary>
+    public class AlipayAuthenticationOptions : OAuthOptions
+    {
+        public string AlipayPublicKey { get; set; }
+        /// <summary>
+        /// Gets or sets a value indicating whether [not validate sign].
```

Comment:
I think it would be clearer if this was named `ValidateSignature` and had a default value of `true`. The `Not` in the name being used to invert the condition isn't very idiomatic C# in my opinion.